test/units/testsuite-07.exec-context.sh

```diff
@@ -119,8 +119,9 @@ if ! systemd-detect-virt -cq; then
         -p IODeviceWeight="/foo/bar 999"
     )
 
+    systemctl set-property system.slice IOAccounting=yes
     # io.latency not available by default on Debian stable
-    if [ -e /sys/fs/cgroup/system.slice/io.latency ]; then
+    if [[ -e /sys/fs/cgroup/system.slice/io.latency ]]; then
```

Comment:
Is `[[` preferred?

Reply:
Since the test scripts are bash scripts I prefer to use the "full" bash syntax (and `[[` is more powerful than just `[`, which is also nice). Similarly with `"x" = "y"` vs `"x" == "y"`).